pegasus/sites.v3/code.org/layouts/wide.haml

```diff
@@ -1,10 +1,5 @@
 .wrapper
-  -if header['theme'] == 'responsive_wide'
-    #desktop-header.desktop-feature
-      -#= view :header
-    #mobile-header.mobile-headers.mobile-feature
-      -#= view :mobile_header_responsive
-  -else
+  -unless header['theme'] == 'responsive_wide'
```

Comment:
Not having any context, I'm going to ask silly questions... The deletion just removes some unused DOM elements, right?

Is there any associated CSS that can also be removed (haven't checked myself)? Can the rest of the page logic (also having gating on 'responsive_wide') be simplified as part of this change?